src/rnnlm/rnnlm-core-training.cc

```diff
@@ -192,6 +192,82 @@ void RnnlmCoreTrainer::Train(
   num_minibatches_processed_++;
 }
 
+void RnnlmCoreTrainer::TrainBackstitch(
+    bool is_backstitch_step1,
```

Comment:
why 1?

Reply:
1 means the 1st step in the backstitch training.